### ISSUE TITLE
Documentation bug in the list of possible <span/> classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2514]
|<img alt="edwards's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| edwards reported|
|-|-|
|Trac ID|trac#2514|
|Type|defect|
|Created|2007-03-09 14:30:17Z|
|Branch|https://github.com/twisted/twisted/tree/span-classes-2514|

In "Elements and their uses", the "span" element is listed as having a "span" class. e.g.

  <span class="span" value="a!b!c"/>

Shouldn't that be "index"?

  <span class="index" value="a!b!c"/>

<details><summary>Searchable metadata</summary>

```
trac-id__2514 2514
type__defect defect
reporter__edwards edwards
priority__normal normal
milestone__ 
branch__branches_span_classes_2514 branches/span-classes-2514
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation documentation
time__1173450617000000 1173450617000000
changetime__1222270372000000 1222270372000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

Indeed it should.  This document has some other problems as well.  For example, the formatting of the information of what attributes have special meaning for what elements is terrible.  Furthermore, the information is not as accurate as it could be.  For example, _any_ element can have the index attribute, not just spans.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [24884]) Branching to 'span-classes-2514'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Moved the elements into a table which is easier to read. Couldn't find the other issues described here, putting it up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

The other issue is still present: the `span` element is documented as being allowed to have a `span` attribute which results in some interaction with the resulting term index.  The correct attribute name for this is `index`.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra removed owner|
|-|-|

Replying to [exarkun](#note_5):
> The other issue is still present: the `span` element is documented as being allowed to have a `span` attribute which results in some interaction with the resulting term index.  The correct attribute name for this is `index`.

Ah so it was that simple :) Fixed in r24887

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

There's some trailing whitespace in the new table markup.  Otherwise this looks good.  Please strip the whitespace and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24915]) Merge span-classes-2514: Correct documentation bug in the list of possible <span/> classes ('index' instead of 'span'). Moved elements into table.

Author: thijs
Reviewer: exarkun
Fixes: [#2514](https://github.com/twisted/twisted/issues/2514)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
